### PR TITLE
F

### PR DESCRIPTION
**Pull request by Vroo**
Implement #362.
I don't know what the pattern is for singletons in this project. I'm using `MailHelper.initialize` and `MailHelper.Singleton`. Happy to change that to whatever is the standard. I couldn't find an existing use case to copy.

### CODE REVIEW

File: ServerCore/Helpers/MailHelper.cs [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        {
+            foreach (string recipient in recipients) {
+                msg.To.Add(new MailAddress(recipient));
+            }
+        }
+
+        msg.Subject = subject;
+        msg.Body = body;
+
+        SmtpClient client = new SmtpClient("in.mailjet.com", 587);
```
Comment (tabascq) [inline ServerCore/Helpers/MailHelper.cs]: Is this specific choice of port something we need to document? No worries if not. I know there are multiple SMTP ports but I do not know enough to know the differences between them.

File: ServerCore/Pages/Hints/Edit.cshtml [hunk 2]
```diff
@@ -13,6 +13,10 @@
     <a asp-page="./Index">Cancel</a>
 </div>
 
+<div>
```
Comment (tabascq) [inline ServerCore/Pages/Hints/Edit.cshtml]: Not sure why these hint things are part of this PR, but if you feel they're truly ready to go I won't quibble, in the name of throughput.

File: ServerCore/Pages/Teams/Hints.cshtml [hunk 3]
```diff
@@ -1,6 +1,48 @@
...
+        color: #aaaaaa;
+    }
+
+    /*
+     * Strikeout style for hints
+     */
+    .strike1 {
+        position: relative;
+    }
+    .strike1:before {
```
Comment (tabascq) [inline ServerCore/Pages/Teams/Hints.cshtml]: I could have sworn I saw and reviewed these before...

Comment (tabascq) [reply ServerCore/Pages/Teams/Hints.cshtml]: Indeed, I have them in my copy of the file. What happened? Did this get reverted somehow and you are replacing it?

**Comment by tabascq** [comment]
FWIW, I think I see many of the hint changes in the master branch when viewed from the web site, which makes me extra baffled why they show up as diffs in the PR.

File: ServerCore/Pages/Hints/Edit.cshtml (hunk 2 above)
Comment (Vroo) [reply ServerCore/Pages/Hints/Edit.cshtml]: There are no changes related to hints. Github confusion. I will fix.

File: ServerCore/Helpers/MailHelper.cs (hunk 1 above)
Comment (Vroo) [reply ServerCore/Helpers/MailHelper.cs]: That's the port that Mailjet tells us to use. I will add a link to their documentation.

Comment (Vroo) [reply ServerCore/Helpers/MailHelper.cs]: Actually, I have no idea how to fix. I will cherry-pick the commit over to a new branch and send out a new review after addressing the issues.

**Pull request closed without merging**